R/zchunk_LB125.LC_tot.R

```diff
@@ -65,38 +64,132 @@ module_aglu_LB125.LC_tot_DISABLED <- function(command, ...) {
     #
     # ===================================================
 
+    # -----------------------------------------------------------------------------
+    # 2. Perform computations
+    #Create a table with all land types
+    #NOTE: If protected lands are included, uncomment the text below
```

Comment:
What lines does this NOTE refer to? 
Also, I'm betting Ben will want a space after each "#".
For example: `# Create a table with all land types` instead of `#Create a table with all land types`.